node/src/chain_spec.rs

```diff
@@ -178,6 +180,7 @@ pub fn local_testnet_config() -> ChainSpec {
 
 fn testnet_genesis(
 	invulnerables: Vec<(AccountId, AuraId)>,
+	root_key: AccountId,
```

Comment:
The root_key param to was added to this function, so "get_account_id_from_seed" call above is how to set the root key, i.e. the sudoer.

Reply:
I think we are going to need to have a pre-prod checklist and I think removing `Alice` or replacing it with else would be in that list.

Reply:
We should have such a checklist anyway. Someone could copy the chain spec to a production one and forget about the sudo.  I set testnet root key to 'None' and regenerated the specs.